Test/GitHubRelease.Tests.ps1

```diff
@@ -0,0 +1,573 @@
+            }
+        }
+
+        if( $UsedMethod )
+        {
+            Assert-MockCalled -CommandName 'Invoke-RestMethod' -ModuleName 'Whiskey' -ParameterFilter { 
+                #$DebugPreference = 'Continue'
+                Write-Debug ('Uri     expected  {0}' -f $ToUri)
+                Write-Debug ('        actual    {0}' -f $Uri)
+                Write-Debug ('Method  expected  {0}' -f $UsedMethod)
+                Write-Debug ('        actual    {0}' -f $Method)
+                $Uri -eq [uri]$ToUri -and $Method -eq $UsedMethod 
+            }
+        }
+
+        if( $AsContentType )
+        {
+            Assert-MockCalled -CommandName 'Invoke-RestMethod' -ModuleName 'Whiskey' -ParameterFilter { 
+                #$DebugPreference = 'Continue'
+                Write-Debug ('Uri          expected  {0}' -f $ToUri)
+                Write-Debug ('             actual    {0}' -f $Uri)
+                Write-Debug ('ContentType  expected  {0}' -f $AsContentType)
+                Write-Debug ('             actual    {0}' -f $ContentType)
+                $Uri -eq [uri]$ToUri -and $ContentType -eq $AsContentType 
+            }
+        }
+
+        if( $WithFile )
+        {
+            Assert-MockCalled -CommandName 'Invoke-RestMethod' -ModuleName 'Whiskey' -ParameterFilter { 
+                #$DebugPreference = 'Continue'
+                Write-Debug ('Uri     expected  {0}' -f $ToUri)
+                Write-Debug ('        actual    {0}' -f $Uri)
+                $WithFile = Join-Path -Path $TestDrive.FullName -ChildPath $WithFile
+                Write-Debug ('InFile  expected  {0}' -f $WithFile)
+                Write-Debug ('        actual    {0}' -f $InFile)
+                $Uri -eq [uri]$ToUri -and $InFile -eq $WithFile 
+            }
+        }
+    }
+}
+
+function ThenSecurityProtocol
+{
+    param(
+        $HasFlag
+    )
+
+    It ('should enable Tls12'){
+        [System.Net.ServicePointManager]::SecurityProtocol.HasFlag($HasFlag) | Should -Be $true
+    }
+}
+
+function ThenTaskFailed
+{
+    It ('should fail') {
+        $failed | Should -Be $true
+    }
+}
+
+function WhenRunningTask
+{
+    [CmdletBinding()]
+    param(
+        $OnCommit
+    )
+
+    $Global:Error.Clear()
+
+    $script:failed = $false
+    try
+    {
+        [Whiskey.Context]$context = New-WhiskeyTestContext -ForDeveloper -ConfigurationPath (Join-Path -Path $TestDrive.FullName -ChildPath 'whiskey.yml')
+        foreach( $key in $apiKeys.Keys )
+        {
+            Add-WhiskeyApiKey -Context $context -ID $key -Value $apiKeys[$key]
+        }
+        $parameter = $context.Configuration['Build'] | Where-Object { $_.ContainsKey('GitHubRelease') } | ForEach-Object { $_['GitHubRelease'] }
+        if( $OnCommit )
+        {
+            $parameter['Commitish'] = $OnCommit
+        }
+        Invoke-WhiskeyTask -TaskContext $context -Name 'GitHubRelease' -Parameter $parameter
+        $script:context = $context

```

Comment:
Setting a global script variable that is never used.

Reply:
Fixed.